tests/integration/test_cli.py

```diff
@@ -60,7 +75,11 @@ def test_multipart_download(
     part_size: int,
     s3_fixture: S3Fixture,  # noqa F811
     tmp_path: pathlib.Path,
+    monkeypatch,
 ):
+    # The download function will ask the user for input.
+    monkeypatch.setattr("builtins.input", mock_input_wps_string)
```

Comment:
There are other places that expect input that this might eventually break in the batch_processing module as everything that is not "yes" is interpreted as no by them
Might be better to factor input handling out into its own function and monkeypatch that instead